package.json

```diff
@@ -65,6 +65,7 @@
     "@discordjs/opus": "^0.5.0",
     "@types/node": "^14.14.41",
     "@types/ws": "^7.4.1",
+    "discord-api-types": "^0.18.1",
```

Comment:
What's the point of adding this dependency? Seems like it's not used.

Reply:
it caused some TypeScript error xD 🙄

Reply:
looks like an issue with the language server

Reply:
What do you mean? 🤔 
It compiles well without d-api-types btw ✅